sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -310,6 +315,14 @@ def send(self, message):
                 :caption: Send message.
 
         """
+        try:
+            batch = self.create_batch()
+            for each in message:
```

Comment:
The method signature is a List. But "for" works for a superset of List. Let's discuss offline. EventHub has a similar case

Reply:
Yes, this was somewhat intentional, I wanted to accept anything enumerable and make a best effort at consuming it.  We could be more restrictive but the general principle seems to be "be as permissive as you can."  Will ping.

Reply:
Per discussion with anna, we're going to keep the document-list accept-permissively approach.  This would seem to give us the most flexibility to either constrain or open behavior in the future.